app/src/main/java/com/truongdc21/quickquotes/Presenter/ActivityViewPlay/ViewPlayActivityPresenter.kt

```diff
@@ -1,4 +1,119 @@
 package com.truongdc21.quickquotes.presenter.activityViewPlay
 
-class ViewPlayActivityPresenter() {
+import android.content.Context
+import android.util.Log
+import com.truongdc21.quickquotes.R
+import com.truongdc21.quickquotes.data.model.Quotes
+import com.truongdc21.quickquotes.data.repository.QuotesRepository
+import com.truongdc21.quickquotes.data.source.local.OnLocalResultListener
+import com.truongdc21.quickquotes.data.source.remote.OnRemoteResultListener
+import java.lang.Exception
+
+class ViewPlayActivityPresenter(
+    private val mContext: Context,
+    private val mRepoQuotes : QuotesRepository
+    ):ViewPlayActivityContract.Presenter{
+
+    private var mView : ViewPlayActivityContract.View? = null
+    private var mListQuotes = mutableListOf<Quotes>()
+    private var mPosition : Int? = null
+
+    override fun onStart() {}
+
+    override fun onStop() {}
+
+    override fun setView(view: ViewPlayActivityContract.View?) {
+       this.mView = view
+    }
+
+    override fun getListQuotesAPIRandom() {
+        mRepoQuotes.getApiQuotesList(object : OnRemoteResultListener<List<Quotes>>{
+            override fun onSuccess(data: List<Quotes>) {
+                mView?.showAdapterAPIRandom(data)
+                mListQuotes.apply {
+                    clear()
+                    addAll(data)
+                }
+            }
+
+            override fun onError(exception: Exception?) {
+                mView?.onError(mContext.resources.getString(R.string.get_list_error))
+            }
+        })
+    }
+
+    override fun setPositionItemQuotes(position: Int) {
```

Comment:
```suggestion
    override fun getItemQuoteByPosition(position: Int) {
```
đây là method để get item đúng ko nhỉ